src/components/services-view/services-view.controller.ts

```diff
@@ -71,6 +71,14 @@ export class ServicesViewController implements angular.IController {
       }
     };
 
+    this.ctrl.noItemsConfig = {
+        title: 'No catalog items have been loaded.',
```

Comment:
Sorry.  I think you still want the title to be "No items." with the info being "No catalog items have been loaded." followed by the help link.  

Another example for reference:  https://user-images.githubusercontent.com/895728/31667665-20f0674a-b31e-11e7-8e83-8b0a047220ba.png

Reply:
Ok, updated to:
![image](https://user-images.githubusercontent.com/12733153/35448503-7672c1cc-0288-11e8-8957-850694d4b309.png)